packages/SwingSet/test/test-promises.js

```diff
@@ -251,24 +249,36 @@ test('local promises are rejected by vat upgrade', async t => {
     const err = c.kpResolution(kpid);
     throw kunser(err);
   };
-  const messageVat = (name, methodName, args) =>
-    run('messageVat', [{ name, methodName, args }]);
-  // eslint-disable-next-line no-unused-vars
-  const messageObject = (presence, methodName, args) =>
-    run('messageVatObject', [{ presence, methodName, args }]);
+
+  const messageToVat = async (vatName, method, ...args) => {
+    assert(Array.isArray(args));
+    const kpid = c.queueToVatRoot(vatName, method, args);
+    return awaitRun(kpid);
+  };
+  const messageToObject = async (presence, method, ...args) => {
+    assert(Array.isArray(args));
+    const kpid = c.queueToVatObject(presence, method, args);
+    return awaitRun(kpid);
+  };
```

Comment:
I don't think `assert(Array.isArray(restParam))` carries its weight.
```suggestion
  const messageToVat = async (vatName, method, ...args) => {
    const kpid = c.queueToVatRoot(vatName, method, args);
    return awaitRun(kpid);
  };
  const messageToObject = async (presence, method, ...args) => {
    const kpid = c.queueToVatObject(presence, method, args);
    return awaitRun(kpid);
  };
```

Reply:
Oh yeah.  I think that's a bit of bleed over from the earlier version of the code, in which `args` was an explicit array parameter rather than using spread.